src/context/WalletProvider/WalletProvider.tsx

```diff
@@ -481,10 +479,11 @@ export const WalletProvider = ({ children }: { children: React.ReactNode }): JSX
                 const localKeepKeyWallet = await (async () => {
                   const maybeWallet = state.keyring.get(localWalletDeviceId)
                   if (maybeWallet) return maybeWallet
-                  const keepKeyAdapters = state.adapters?.get(KeyManager.KeepKey)
-                  if (!keepKeyAdapters) return
+                  const keepKeyAdapter = await getAdapter(KeyManager.KeepKey)
+                  if (!keepKeyAdapter) return
                   const sdk = await setupKeepKeySDK()
-                  return await keepKeyAdapters[0]?.pairDevice(sdk)
+                  // @ts-ignore TODO(gomes): FIXME, most likely borked because of WebUSBKeepKeyAdapter
```

Comment:
WebUSBKeepKeyAdapter types are too borked, fiddled for two hours in hdwallet with no success here, leaving this as-is for now - there is no effective runtime change here